commons/logging/src/main/java/org/operaton/commons/logging/BaseLogger.java

```diff
@@ -276,4 +281,25 @@ protected String exceptionMessage(String id, String messageTemplate, Object... p
     }
   }
 
+  /**
+   *  Sanitize parameters to avoid injection attacks
+   */
+  private Object[] sanitizeParameters(Object... parameters) {
+    if (parameters == null) {
+      return new Object[0];
```

Comment:
This is modifying the behavior. Before this change, null was passed to the logger, now it is an empty array. While null should not be passed on to the logger, I don't see any reason why this change should be the responsibility of `sanitizeParameters`.

Reply:
good catch. I have changed that to return `null` and added a test for this method. However, it leaves a warning that an empty array should be returned instead of null. Added a suppression for it.